### ISSUE TITLE
Can I make it so the canvas adjust to the size of the div he's in?

### ISSUE THREAD

**Issue by fmarlete** [alexk111/ngImgCrop#57]
So I'd like to know if it's possible to force the canvas to be have the same height and width than the div he's in?

Thanks =)

**Comment by coffeeboss** [comment]
Yes, I suggest you put a max-width\max-height at the parent div of ngimgcrop and let your framework (Bootstrap, Foundation, etc.) do the heavy lifting for you.
